node/src/Pos/Launcher/Runner.hs

```diff
@@ -122,8 +124,8 @@ runRealModeDo NodeResources {..} outSpecs action =
         case npEnableMetrics of
             False -> return Nothing
             True  -> Just <$> do
-                ekgStore' <- setupMonitor
-                    (runProduction . runToProd JsonLogDisabled oq) node' nrEkgStore
+                let ekgStore' = nrEkgStore
+                registerMetrics (Just cardanoNamespace) (runProduction . runToProd JsonLogDisabled oq) node' ekgStore'
```

Comment:
You don't need `ekgStore'` any more. It was only there because the old `setupMonitor` returned a possibly modified store, and no it doesn't, so there's no need to name the result.